scripts/pre_leap.sh

```diff
@@ -20,9 +20,8 @@ set -e -u
 set -o pipefail
 
 # Preserve Elasticsearch data in the leap
-echo 'Elasticsearch logs will be preserved.'
-export UPGRADE_ELASTICSEARCH="YES"
-export CONTAINERS_TO_DESTROY='all_containers:!galera_all:!neutron_agent:!ceph_all:!rsyslog_all:!elasticsearch_all'
+export UPGRADE_ELASTICSEARCH={$UPGRADE_ELASTICSEARCH:-"yes"}
+export CONTAINERS_TO_DESTROY={$CONTAINERS_TO_DESTROY:-'all_containers:!galera_all:!neutron_agent:!ceph_all:!rsyslog_all:!elasticsearch_all'}
```

Comment:
Should look like these, dollar sign should be outside the bracket and probably need the quotes set like we did upstream to prevent bash from executing the !s:

export UPGRADE_ELASTICSEARCH="${UPGRADE_ELASTICSEARCH:-YES}"
export CONTAINERS_TO_DESTROY='"'"${CONTAINERS_TO_DESTROY:-all_containers:!galera_all:!neutron_agent:!ceph_all:!rsyslog_all:!elasticsearch_all}"'"'

Reply:
Cool, will correct them.